lib/Perl/Critic/Policy/ControlStructures/ProhibitYadaOperator.pm

```diff
@@ -0,0 +1,95 @@
+package Perl::Critic::Policy::ControlStructures::ProhibitYadaOperator;
+
+use 5.006001;
+use strict;
+use warnings;
+use Readonly;
+
+use Perl::Critic::Utils qw{ :characters :severities };
+use base 'Perl::Critic::Policy';
+
+our $VERSION = '1.123';
+
+#-----------------------------------------------------------------------------
+
+Readonly::Scalar my $DESC => q{yadda operator (...) used};
+Readonly::Scalar my $EXPL => [ 100 ];
```

Comment:
What does the `100` mean? For PBP polices, the `$EXPL` are page numbers. For non-PBP the `$EXPL` is usually used as a longer description of the violation.